src/balena.sbvr

```diff
@@ -60,6 +60,8 @@ Term: api key
 Fact type: api key has key
 	Necessity: each api key has exactly one key
 	Necessity: each key is of exactly one api key
+Fact type: api key has expiry date
```

Comment:
@Page- Is it necessary to add `Fact Type` defn here as well, given we already have one in `pinejs` (https://github.com/balena-io/pinejs/pull/538/files#diff-5d319ba52d1182a4a33085150eb61774e956cf7380813686a02ed0d508f6cc00R54)?

Reply:
Yes it is if you want to be able to do `resinApi.post` to `user`, since it only looks at this vocab - it's ugly but necessary for the time being

Reply:
okay. @Page- what I can start to look to understand the current wiring/logic?